e2e_playwright/st_dialog_test.py

```diff
@@ -180,14 +184,18 @@ def test_largewidth_dialog_displays_correctly(
     open_largewidth_dialog(app)
     wait_for_app_run(app)
     dialog = app.get_by_role("dialog")
-    expect(dialog.get_by_test_id("stButton")).to_be_visible()
+    # click on the dialog title to take away focus of all elements and make the screenshot stable. Then hover over the button for visual effect.
+    dialog.locator("div", has_text="Large-width Dialog").click()
+    submit_button = dialog.get_by_test_id("stButton")
+    expect(submit_button).to_be_visible()
+    submit_button.get_by_test_id("baseButton-secondary").hover()
```

Comment:
`submit_button.hover()` doesn't work, and we need a more narrow locator here?

Reply:
yep, I tried the other one first but apparently the effect happens on the button element itself!